### ISSUE TITLE
Cannot Create Route53 Record

### ISSUE THREAD

**Issue by smslattery** [localstack/localstack#3221]
# Type of request: This is a ...
[x] bug report
[ ] feature request

# Detailed description
When attempting to create an `aws_route53_record` resource using terraform, it is failing to create the resource, entering the `"Still creating..."` loop.

## Expected behavior
The defined resources should be created.

## Actual behavior
The `aws_route53_record` fails to be created (see full recap below).

# Steps to reproduce
## Command used to start LocalStack
`IMAGE_NAME=localstack/localstack-full:0.12.1 localstack start`

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)
This was executed using terraform v0.12.6

```
$ terraform apply -auto-approve
aws_route53_zone.example: Creating...
aws_instance.example: Creating...
aws_route53_zone.example: Creation complete after 0s [id=A1QUMZB6FOKZ3ZU]
aws_instance.example: Still creating... [10s elapsed]
aws_instance.example: Creation complete after 10s [id=i-cafc5cc4c945c2fec]
aws_eip.example: Creating...
aws_eip.example: Creation complete after 0s [id=eipalloc-f42d6854]
aws_route53_record.example: Creating...
aws_route53_record.example: Still creating... [10s elapsed]
aws_route53_record.example: Still creating... [20s elapsed]
aws_route53_record.example: Still creating... [30s elapsed]
aws_route53_record.example: Still creating... [40s elapsed]
aws_route53_record.example: Still creating... [50s elapsed]
aws_route53_record.example: Still creating... [1m0s elapsed]
aws_route53_record.example: Still creating... [1m10s elapsed]
aws_route53_record.example: Still creating... [1m20s elapsed]
aws_route53_record.example: Still creating... [1m30s elapsed]
aws_route53_record.example: Still creating... [1m40s elapsed]
aws_route53_record.example: Still creating... [1m50s elapsed]
aws_route53_record.example: Still creating... [2m0s elapsed]
aws_route53_record.example: Still creating... [2m10s elapsed]
^CInterrupt received.
Please wait for Terraform to exit or data loss may occur.
Gracefully shutting down...
Stopping operation...
^CTwo interrupts received. Exiting immediately. Note that data
loss may have occurred.

Error: operation canceled


$ terraform destroy -auto-approve
aws_route53_zone.example: Refreshing state... [id=A1QUMZB6FOKZ3ZU]
aws_instance.example: Refreshing state... [id=i-cafc5cc4c945c2fec]
aws_eip.example: Refreshing state... [id=eipalloc-f42d6854]
aws_route53_zone.example: Destroying... [id=A1QUMZB6FOKZ3ZU]
aws_eip.example: Destroying... [id=eipalloc-f42d6854]
aws_route53_zone.example: Destruction complete after 0s
aws_eip.example: Destruction complete after 0s
aws_instance.example: Destroying... [id=i-cafc5cc4c945c2fec]
aws_instance.example: Still destroying... [id=i-cafc5cc4c945c2fec, 10s elapsed]
aws_instance.example: Destruction complete after 10s

Destroy complete! Resources: 3 destroyed.
```

## Terraform Config Files
main.tf

```
resource "aws_route53_zone" "example" {
  name = "test.example.com"
  tags = {
    name = "some-aws-route53-zone"
  }
}

resource "aws_instance" "example" {
  ami           = "ami-5189a661"
  instance_type = "t2.micro"
  tags = {
    name = "some-instance"
  }
}

resource "aws_eip" "example" {
  vpc = true
  instance = aws_instance.example.id
  tags = {
    name = "some-eip"
  }
}

resource "aws_route53_record" "example" {
  zone_id = aws_route53_zone.example.zone_id
  name    = "test.example.com"
  type    = "A"
  ttl     = "5"
  records = [aws_eip.example.public_ip]
}
```

outputs.tf

```
output "aws_route53_zone" {
  value       = aws_route53_zone.example
}

output "aws_instance" {
  value       = aws_instance.example.tags
}

output "aws_eip" {
  value       = aws_eip.example
}

output "aws_route53_record" {
  value       = aws_route53_record.example
}
```

site.tf

```
provider "aws" {
  region                      = "us-east-1"
  access_key                  = "mock_access_key"
  secret_key                  = "mock_secret_key"
  s3_force_path_style         = true
  skip_credentials_validation = true
  skip_metadata_api_check     = true
  skip_requesting_account_id  = true
  endpoints {
    ec2            = "http://localhost:4566"
    route53        = "http://localhost:4566"
  }
}
```

┆Issue is synchronized with this [Jira Task](https://localstack.atlassian.net/browse/LOC-64) by [Unito](https://www.unito.io/learn-more)

**Comment by smslattery** [comment]
This may be related to issue https://github.com/localstack/localstack/issues/1860
However, since that ticket was almost a year old, I added this one with simple example reproduction steps.
I also saw that the ticket above had a `PRO` tag on it, but `route53` is listed under the free usage components in the README.

**Comment by macnev2013** [comment]
Thanks for reporting @smslattery. This should be fixed in #3248 - can you please give it another try with the latest version of the Docker image? Please report here if the problem persists. Thanks

**Comment by smslattery** [comment]
Confirmed with latest as of today.
